src/v/features/feature_table.h

```diff
@@ -153,6 +154,12 @@ constexpr static std::array feature_schema{
     feature::raftless_node_status,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{6},
```

Comment:
This will be version 7 -- if you want to get ahead of a rebase later see the "increment cluster version" commit in https://github.com/redpanda-data/redpanda/pull/6529/commits (raftless_node_status should also have been version 7)